dbt_subprojects/hourly_spellbook/models/_project/cow_protocol/ethereum/cow_protocol_ethereum_solvers.sql

```diff
@@ -112,6 +112,8 @@ known_solver_metadata (address, environment, name) as (
                  (0xc10D4DfDA62227d9EC23Ab0010E2942e48338A60, 'prod', 'Apollo'),
                  (0x008300082C3000009e63680088f8c7f4D3ff2E87, 'prod', 'Copium_Capital'),
                  (0xC7899Ff6A3aC2FF59261bD960A8C880DF06E1041, 'prod', 'Barter'),
+                 (0x8A75ee64b4A40f679aD98Bcc38312702971e07B7, 'prod', 'OneBit Quant'),
```

Comment:
I would avoid using spaces and would add an underscore instead, as this has been the convention so far